Source/WebKit/NetworkProcess/soup/NetworkDataTaskSoup.cpp

```diff
@@ -1436,10 +1436,15 @@ void NetworkDataTaskSoup::didGetFileInfo(GFileInfo* info)
         m_response.setExpectedContentLength(-1);
     } else {
         auto contentType = String::fromLatin1(g_file_info_get_content_type(info));
-        m_response.setMimeType(extractMIMETypeFromMediaType(contentType));
+        auto mimeTypeFromExtension = MIMETypeRegistry::mimeTypeForPath(m_response.url().path().toString());
```

Comment:
mimeTypeForPath takes StringView after <https://commits.webkit.org/248480@main>. We can remove `.toString()` here.